components/network/ntp/patches/70-warrnigs.patch

```diff
@@ -0,0 +1,62 @@
+--- util/ntptime.c.orig	2020-03-04 00:41:29.000000000 +0100
```

Comment:
Please add a comment explaining why we do need this.  Thank you.

Reply:
BTW, the patch filename contains typo (two `r`, missing `n`).

Reply:
removed